en: update get-started.md for 2.0 (#52)
pingcap/docs-dm: en/get-started.md

## Setup

We're going to deploy 3 instances of MySQL Server, and 1 instance each of pd-server, tikv-server, and tidb-server. Then we'll start 3 instances of DM-master and 3 instances of DM-worker.

> 1. Install MySQL 5.7, download and extract the TiDB v4.0 and DM v2.0 packages we'll use:

Reviewer: I think they are two separate sentences? We should use a period or semicolon to denote that.
```suggestion
1. Install MySQL 5.7. Then download and extract the TiDB v4.0 and DM v2.0 packages we'll use:
```

> 

Reviewer: ```suggestion

    {{< copyable "shell-regular" >}}

```

    [3]   Running                 mysqld --defaults-group-suffix="$i" --user=root &
    [4]   Running                 tidb-server --log-file=logs/tidb-server.log &
    [5]   Running                 dm-master --config=dm-cnf/dm-master"$i".toml --log-file=logs/dm-master"$i".log >> logs/dm-master"$i".log 2>&1 &
    [6]   Running                 dm-master --config=dm-cnf/dm-master"$i".toml --log-file=logs/dm-master"$i".log >> logs/dm-master"$i".log 2>&1 &
    [7]   Running                 dm-master --config=dm-cnf/dm-master"$i".toml --log-file=logs/dm-master"$i".log >> logs/dm-master"$i".log 2>&1 &
    [8]   Running                 dm-worker --config=dm-cnf/dm-worker"$i".toml --log-file=logs/dm-worker"$i".log >> logs/dm-worker"$i".log 2>&1 &
    [9]-  Running                 dm-worker --config=dm-cnf/dm-worker"$i".toml --log-file=logs/dm-worker"$i".log >> logs/dm-worker"$i".log 2>&1 &
    [10]+  Running                 dm-worker --config=dm-cnf/dm-worker"$i".toml --log-file=logs/dm-worker"$i".log >> logs/dm-worker"$i".log 2>&1 &
    ```

    {{< copyable "shell-regular" >}}

    ```bash
    ps -a
    ```

>     ```

Reviewer: ```suggestion
    ```
       PID TTY          TIME CMD
```

       285 pts/0    00:00:00 dm-worker
       324 pts/0    00:00:00 ps
    ```

3. Load the data source configuration into the DM cluster using dmctl, and execute the following command to get the returned results.

    {{< copyable "shell-regular" >}}

    ```bash
    for i in 1 2 3; 
    do dmctl --master-addr=127.0.0.1:8261 operate-source create dm-cnf/source"$i".toml 
    done
    ```

    ```
    {
        "result": true,
        "msg": "",
        "sources": [
            {
                "result": true,
                "msg": "",
                "source": "mysql-replica-01",
                "worker": "worker1"
            }
        ]
    }
    {
        "result": true,
        "msg": "",
        "sources": [
            {
                "result": true,
                "msg": "",
                "source": "mysql-replica-02",
                "worker": "worker2"
            }
        ]
    }
    {
        "result": true,
        "msg": "",
        "sources": [
            {
                "result": true,
                "msg": "",
                "source": "mysql-replica-03",
                "worker": "worker3"
            }
        ]
    }
    ```

> Each of the upstream MySQL Server instances corresponds to a separate DM-worker instance, each of which has its own configuration file.

Reviewer: ```suggestion
Each of the upstream MySQL Server instances corresponds to a separate `source.toml`. Each DM-master and DM-worker instance has its own configuration file.
```

The following is an example of `dm-master1.toml`:

{{< copyable "" >}}

```toml
# DM-master1 Configuration.

name = "master1"
master-addr = ":8261"
advertise-addr = "127.0.0.1:8261"
peer-urls = "127.0.0.1:8291"
initial-cluster = "master1=http://127.0.0.1:8291,master2=http://127.0.0.1:8292,master3=http://127.0.0.1:8293"
```

> The following is an example of `dm-worker1.toml`:

Reviewer: ```suggestion
The following is an example of `dm-worker1.toml`:

{{< copyable "" >}}
```

```toml

> # Worker Configuration.

Reviewer: ```suggestion
# DM-worker1 Configuration.
```

Our goal in this exercise is to use DM to combine the data from these distinct MySQL instances into a single table in TiDB.

> The package of configuration files we unpacked earlier (dm-cnf.tgz) contains the configuration for the components of the TiDB cluster, the DM components, and for a DM task we'll explore in this tutorial.

Reviewer: ```suggestion
The package of configuration files we have unpacked earlier (dm-cnf.tgz) contains the configuration for the components of the TiDB cluster, the DM components, and for a DM task we'll explore in this tutorial.
```

* We use `black-white-list` to limit the scope of this task to database `dmtest`.

> * The `loaders` section defines where to find the output of each instance of Mydumper that was executed by the respective MySQL source.

Reviewer: ```suggestion
* The `loaders` section defines where to find the output of each instance of Mydumper that has been executed by the respective MySQL source.
```
